src/features/call/components/Survey/SurveyOptions.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+  SelectChangeEvent,
+  Typography,
+} from '@mui/material';
+import { FC, useState } from 'react';
+
+import messageIds from 'features/surveys/l10n/messageIds';
+import { useMessages } from 'core/i18n';
+import {
+  ZetkinSurveyOption,
+  ZetkinSurveyOptionsQuestionElement,
+} from 'utils/types/zetkin';
+import SurveyContainer from 'features/surveys/components/surveyForm/SurveyContainer';
+import SurveySubheading from 'features/surveys/components/surveyForm/SurveySubheading';
+import SurveyQuestionDescription from 'features/surveys/components/surveyForm/SurveyQuestionDescription';
+
+export type OptionsQuestionProps = {
+  element: ZetkinSurveyOptionsQuestionElement;
+  onChange: (value: number | number[]) => void;
+};
+
+const SurveyOptions: FC<OptionsQuestionProps> = ({ element, onChange }) => {
```

Comment:
I don't think this component is necessary, as the same kind of UI is already covered by  `SurveyOptionsQuestion`. Correct me if I'm missing something and this needs to be duplicated for some reason.

Reply:
Correct. Now that I'm using `<SurveyElements />` I don't it anymore. I'll delete it.